src/filters/index.js

```diff
@@ -61,7 +61,7 @@ var digitsRE = /(\d{3})(?=\d)/g
 exports.currency = function (value, currency) {
   value = parseFloat(value)
   if (!isFinite(value) || (!value && value !== 0)) return ''
-  currency = currency || '$'
+  currency = currency === false ? '' : currency || '$'
```

Comment:
Hmm, in suggestion, OP needs ability to filter number through currency filter *without adding symbol*. How would `currency = currency != null ? currency : '$'` do that?